internal/kubernetes/cert.go

```diff
@@ -47,8 +47,8 @@ func (v KubeletCertificateValidator) Run(ctx context.Context, informer validatio
 	defer func() {
 		informer.Done(ctx, name, err)
 	}()
-	if err = hybrid.ValidateCertificate(v.certPath, cluster.CertificateAuthority); err != nil {
-		err = hybrid.AddKubeletRemediation(v.certPath, err)
+	if err = certificate.Validate(v.certPath, cluster.CertificateAuthority); err != nil {
+		err = certificate.AddKubeletRemediation(v.certPath, err)
 		return err
```

Comment:
nit
```suggestion
		return certificate.AddKubeletRemediation(v.certPath, err)
```

Reply:
We need this so the informer has the decorated validation error which includes the remediation, instead of just just the error alone.